src/api/main.py

```diff
@@ -209,5 +213,31 @@ def create_simplify_tolerance_by_area(shp_file_path):
     return floor(linear_interpolate(min(1, area / MAX_AREA_KM2), 1, 2000))
 
 
+def adjust_coord_precision(features, precision=5):
+    """
+    Returns features with coordinates rounded to n digits of precision.
+    Adapted from https://github.com/perrygeo/geojson-precision
+
+    Keyword arguments:
+    features  -- a list of features
+    precision -- the number of digits to round (default 5)
+    """
+    def round_to_precision(n):
+        if type(n) is list:
+            return [round_to_precision(f) for f in n]
+        elif type(n) is float:
+            return round(n, int(precision))
+
+    def round_point(pt):
+        return [round_to_precision(n) for n in pt]
```

Comment:
It occurred to me that this helper's probably superfluous since `round_to_precision`'s list case has the same implementation. It's also probably worth adding a `return n` otherwise case to `round_to_precision`.